frontend/src/app/Settings/Facility/facilitySchema.ts

```diff
@@ -30,7 +34,20 @@ const providerSchema: yup.SchemaOf<RequiredProviderFields> = yup.object({
 
 export const facilitySchema: yup.SchemaOf<RequiredFacilityFields> = yup.object({
   name: yup.string().required(),
-  cliaNumber: yup.string().required(),
+  cliaNumber: yup
```

Comment:
One additional comment: should update the error messages listed in the object below:

```
cliaNumber: "CLIA number is missing",
```

I don't think there's a design specification for this, but we need some way to communicate to users that the CLIA number is incorrectly formatted and showing them what the right format is

Reply:
I made a minor update to the error text to indicate that the CLIA number may be incorrectly formatted (429bdd6).

That change was pretty unambitious but I'm not sure if a more complex error message giving the expected format would require design input. @katiealoisi (or anyone) how do we like to approach things like this on this team?

Reply:
We have another ticket (#1144) to identify specific validation errors, so if there's an invalid data format, we tell the user what the correct format should be instead of making them guess. I think this is super important but we should handle it holistically as part of ticket #1144.

Reply:
Update: discussed on Slack with @katiealoisi and @sparkle-usds and decided to add the more descriptive CLIA validation error message as a part of this work.

Reply:
Added in b994812. @nickscialli-usds please let me know if this resolves your concern.